packages/@react-aria/ssr/src/SSRProvider.tsx

```diff
@@ -10,8 +10,10 @@
  * governing permissions and limitations under the License.
  */
 
-import React, {ReactNode, useContext, useMemo, useState} from 'react';
-import {useLayoutEffect} from '@react-aria/utils';
```

Comment:
isn't this the really important place for it, the whole reason we have a useLayoutEffect hook in react-aria/utils is to fix problems with it in SSR?

Reply:
Did you see the comment below? I think this one is an exception

Reply:
i missed the comment